cinema/serializers.py

```diff
@@ -0,0 +1,18 @@
+from rest_framework import serializers
+from .models import Movie
+
+
+class MovieSerializer(serializers.Serializer):
+    title = serializers.CharField(read_only=True)
+    description = serializers.CharField(required=False, max_length=255)
```

Comment:
The `description` field in the `MovieSerializer` has a `max_length` of 255, which does not match the `TextField` type in the `Movie` model. Consider removing the `max_length` attribute or ensuring it aligns with the model's field type.